texmf/tex/latex/tagpdf/tagpdf-mc-code-lua.sty

```diff
@@ -76,6 +76,8 @@
       }
   }
 \cs_new_protected:Npn \__tag_add_missing_mcs_to_stream:Nn #1#2 {}
+\cs_set_eq:NN \tag_mc_add_missing_to_stream:Nn \__tag_add_missing_mcs_to_stream:Nn
+\cs_new_protected:Npn  \tag_mc_new_stream:n #1 {}
```

Comment:
do you really want to keep both internal and public for longer? I think it should (eventually at least) a replacement